armi/reactor/blueprints/tests/test_componentBlueprint.py

```diff
@@ -110,6 +112,12 @@ def test_componentInitializationControlCustomIsotopics(self):
         for nuc in unexpectedNuclides:
             self.assertNotIn(nuc, a[0][0].getNuclides())
 
+        c = a[0][0]
+        # Watch out, depletion is adding DEPLETABLE, so this can be a bit brittle
```

Comment:
is "depletion" going to make sense in this context?

Reply:
not really... almost as if we shouldn't be sneakily adding the DEPLETABLE flag in the component blueprint in the first place, because it forces us to know about it in this context. Cat's kind of out of the bag, though. The purpose of the comment is to provide a hint to a future developer that tries to change that behavior as to why the test would start breaking.